docs/src/man/basics.md

```diff
@@ -0,0 +1,2170 @@
+  995 │    50   7.07107
+  996 │    31   5.56776
+  997 │    40   6.32456
+  998 │    38   6.16441
+  999 │    23   4.79583
+ 1000 │    27   5.19615
+        985 rows omitted
+```
+
+We can skip specifying a target column name, in which case it is generated automatically by suffixing source
+column name by function name that is applied to it. For example:
+
+```jldoctest dataframe
+julia> select(german, :Sex => ByRow(uppercase))
+1000×1 DataFrame
+  Row │ Sex_uppercase
+      │ String
+──────┼───────────────
+    1 │ MALE
+    2 │ FEMALE
+    3 │ MALE
+    4 │ MALE
+    5 │ MALE
+    6 │ MALE
+    7 │ MALE
+    8 │ MALE
+  ⋮   │       ⋮
+  994 │ MALE
+  995 │ MALE
+  996 │ FEMALE
+  997 │ MALE
+  998 │ MALE
+  999 │ MALE
+ 1000 │ MALE
+      985 rows omitted
+```
+
+`Not()` selector is allowing us to specify column `:Age` we want to exclude from
+the resulting data frame:
+
+```jldoctest dataframe
+julia> select(german, Not(:Age)) 
+1000×9 DataFrame
+  Row │ id     Sex     Job    Housing  Saving accounts  Checking account  Cred ⋯
+      │ Int64  String  Int64  String   String           String            Int6 ⋯
+──────┼─────────────────────────────────────────────────────────────────────────
+    1 │     0  male        2  own      NA               little                 ⋯
+    2 │     1  female      2  own      little           moderate
+    3 │     2  male        1  own      little           NA
+    4 │     3  male        2  free     little           little
+    5 │     4  male        2  free     little           little                 ⋯
+    6 │     5  male        1  free     NA               NA
+    7 │     6  male        2  own      quite rich       NA
+    8 │     7  male        3  rent     little           moderate
+  ⋮   │   ⋮      ⋮       ⋮       ⋮            ⋮                ⋮               ⋱
+  994 │   993  male        3  own      little           little                 ⋯
+  995 │   994  male        2  own      NA               NA
+  996 │   995  female      1  own      little           NA
+  997 │   996  male        3  own      little           little
+  998 │   997  male        2  own      little           NA                     ⋯
+  999 │   998  male        2  free     little           little
+ 1000 │   999  male        2  own      moderate         moderate
+                                                  3 columns and 985 rows omitted
+```
+
+In order to select a column we just passed them as argument. The `r"Sex"` regular expression picks all columns that 
+contain `"Sex"` string in their name:
+
+```jldoctest dataframe
+julia> select(german, r"Sex") 
+1000×1 DataFrame
+  Row │ Sex
+      │ String
+──────┼────────
+    1 │ male
+    2 │ female
+    3 │ male
+    4 │ male
+    5 │ male
+    6 │ male
+    7 │ male
+    8 │ male
+  ⋮   │   ⋮
+  994 │ male
+  995 │ male
+  996 │ female
+  997 │ male
+  998 │ male
+  999 │ male
+ 1000 │ male
+985 rows omitted
+
+julia> select(german, r"Sex", "Job") # Selects more than one column
```

Comment:
what do you mean by "more than one column"? Also do not use comments in code - but split blocks and use markdown.